app/scripts/modules/core/src/pagerDuty/PageModal.tsx

```diff
@@ -7,7 +7,7 @@ import { IPagerDutyService, PagerDutyWriter } from 'core/pagerDuty';
 import { NgReact, ReactInjector } from 'core/reactShims';
 import { SETTINGS } from 'core/config';
 import { SubmitButton } from 'core/modal';
-import { TaskMonitor } from 'core/task';
+import { TaskMonitor, TaskExecutor } from 'core/task';
```

Comment:
doesn't look like `TaskExecutor` is used?

Reply:
Whoops, thanks for catching that. It was residual from a bunch of crap I stuffed in to test this without actually paging people.